packages/nimble-components/src/table/models/sort-operations.ts

```diff
@@ -46,3 +46,41 @@ function localeAwareCaseSensitiveSortFunction<TData>(
     }
     return 1;
 }
+
+/**
+ * A function to perform a basic sort of two rows from TanStack for a given column.
+ * The function sorts `undefined` followed by `null` before all other values that are
```

Comment:
Don't forget to either mention NaN in the comment or change it to be more generic